.dependabot/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 1
+
+update_configs:
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "weekly"
+    default_reviewers:
+    - ryanseddon
```

Comment:
Would it be possible to try the `zendeskgarden/maintainers` group for this? Not sure if it will run into the same permissions issue as Renovate